### ISSUE TITLE
Performance tests

### ISSUE THREAD

**Issue by WojciechNagorski** [ExtendedXmlSerializer/home#279]
I wanted to upgrade BenchmarkDotNet to 0.12.0 version but I don't see `ExtendedXmlSerializer.Tests.Performance` in solution. 

I think that we should have performance tests and we have to add in documentation results of these benchmarks.

**Comment by WojciechNagorski** [comment]
@Mike-EEE What do you think?

**Comment by Mike-EEE** [comment]
Ah, I updated the build script to only build `ExtendedXmlSerializer.sln` and it only contains projects that are used in the build (for test and packaging).  Conveniently, they also contain the projects used to primarily develop. 😁 

I also created an SLN specifically for performance here:

https://github.com/ExtendedXmlSerializer/ExtendedXmlSerializer/blob/master/ExtendedXmlSerializer%20Performance.sln

Likewise, I did the same for samples here:

https://github.com/ExtendedXmlSerializer/ExtendedXmlSerializer/blob/master/ExtendedXmlSerializer%20Samples.sln

BTW I am not too sure our performance is so good these days. 😞
